app/src/main/res/values/strings.xml

```diff
@@ -472,6 +472,7 @@
     <string name="mynotes">My Notes</string>
     <string name="mynote_add">Add My Note</string>
     <string name="mynote_saved">My Note saved</string>
+    <string name="save">Save</string>
```

Comment:
When I was translating I noticed that I was processing a lot of similar messages. 
So, I thought it was better to have only one "save" message, instead of multiple.

Reply:
True, but it is possible that in other languages "save" is translated differently in different contexts. That's why I think it's safer to have repetition in translation strings.